docs/docusaurus/docs/core/set_up_a_gx_environment/set_up_a_gx_environment.md

```diff
@@ -40,4 +40,12 @@ import OverviewCard from '@site/src/components/OverviewCard';
     icon="/img/expectation_icon.svg" 
   />
 
+  <LinkCard 
```

Comment:
**Note for reviewer** this landing page was indexing all pages in the section except for this one